setup.cfg

```diff
@@ -1,5 +1,9 @@
-[nosetests]
-where = deux/tests
+[aliases]
+test=pytest
+
+[tool:pytest]
```

Comment:
I also like to add:
```
addopts = --cov=deux --cov-report term --cov-report html:cover --color yes --no-cov-on-fail
```
Which will show a pretty coverage output, so developers can easily see coverage if all tests pass.